apps/cdd-backend/src/cdd-worker/cdd.processor.ts

```diff
@@ -17,15 +17,23 @@ import {
 } from './types';
 import { Identity } from '@polymeshassociation/polymesh-sdk/types';
 import { NetkiBusinessApplicationModel } from '../app-redis/models/netki-business-application.model';
+import { App as SlackApp } from '@slack/bolt';
 
 @Processor()
 export class CddProcessor {
+  private slackApp: SlackApp;
```

Comment:
This should be its own service, see `AppRedisService` for an example. It should expose a single method to send a message, something like:

```ts
public async sendMessage(title: string, message: string)
```

That way its easy to import into other places and should clean up some boiler plate